shared/ui/Eleitores/Eleitores.jsx

```diff
@@ -0,0 +1,87 @@
+import "/shared/locales/i18n";
```

Comment:
amazing work in here! Lets just create another folder as `components` and move these files there so we can reuse later

Reply:
thanks ❤️